sentry/sentry.conf.example.py

```diff
@@ -3,6 +3,15 @@
 
 from sentry.conf.server import *  # NOQA
 
+UNIT_SIZES={
+    "K": 1024,
+    "M": 1024**2,
+    "G": 1024**3,
+}
+def unit_text_to_bytes(text):
+    multiplier = UNIT_SIZES[text[-1]]
+    return float(text[:-1])*multiplier
```

Comment:
Now that you put my code directly there, I think we can make it even simpler 😅 
```suggestion
BYTE_MULTIPLIER = 1024
UNITS = ("K", "M", "G")
def unit_text_to_bytes(text):
		unit = text[-1].upper()
		power = UNITS.index(unit) + 1
    return float(text[:-1])*(BYTE_MULTIPLIER**power)
```